src/main/webapp/app/exercises/programming/manage/grading/testcase-analysis/testcase-analysis.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpResponse } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { ProgrammingExerciseServerSideTask } from 'app/entities/hestia/programming-exercise-task.model';
+
+export interface SimplifiedTask {
+    taskName: string;
+    testCases: ProgrammingExerciseServerSideTask['testCases'];
+}
+
+export interface FeedbackDetailsWithResultIdsDTO {
+    feedbackDetails: { detailText: string; testCaseName: string }[];
+    resultIds: number[];
+}
```

Comment:
Note that the attributes may become optional (possibly `undefined`) once you add the JSON annotation

Reply:
should be fine, i handle many cases 👍